kuma/sample_apps.go

```diff
@@ -21,3 +25,44 @@ func (kuma *Kuma) installSampleApp(del bool, namespace string, templates []adapt
 
 	return status.Installed, nil
 }
+
+func (kuma *Kuma) sidecarInjection(namespace string, del bool) error {
+	kclient := kuma.KubeClient
+	if kclient == nil {
+		return ErrNilClient
+	}
+
+	// updating the label on the namespace
+	ns, err := kclient.CoreV1().Namespaces().Get(context.TODO(), namespace, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	if ns.ObjectMeta.Labels == nil {
+		ns.ObjectMeta.Labels = map[string]string{}
+	}
+	ns.ObjectMeta.Labels["openservicemesh.io/monitored-by"] = "kuma"
```

Comment:
@sayantan1413 sidecar proxy injection in Kuma Mesh is managed by setting annotation:

```yaml
annotations:
    kuma.io/sidecar-injection: enabled
```

Reply:
Hey @utkarsh-pro I think this is there in line number 54.

Reply:
@sayantan1413 I meant to say that to enable sidecar injection in Kuma mesh we do not need this lable rather the annotation that I have mentioned above. Also annotation on line 54 is for osm.

Reply:
Oh okay so i removed the labels now and updated the link to kuma.io😃

Reply:
Thanks